package.json

```diff
@@ -111,6 +110,9 @@
     "tocbot": "^4.12.2",
     "webpack-merge": "^5.7.3"
   },
+  "peerDependencies": {
```

Comment:
does it need to be a peer dep?

Reply:
https://evilmartians.com/chronicles/postcss-8-plugin-migration

Reply:
![Screen Shot 2021-04-13 at 8 52 40 AM](https://user-images.githubusercontent.com/39598672/114555408-aa579b80-9c35-11eb-8bbf-71e19409db25.png)

Reply:
This is what the guideline recommends.

Reply:
I put it back in the devDeps